test/fixtures.ts

```diff
@@ -0,0 +1,105 @@
+import type fs from 'fs';
+import type { PathLike } from 'fs';
+import callsite from 'callsite';
+import { DirectoryJSON, fs as memfs, vol } from 'memfs';
+import upath from 'upath';
+
+const realFs = jest.requireActual<typeof fs>('fs');
+
+/**
+ * Class to work with in-memory file-system
+ */
+export class Fixtures {
+  /**
+   * Returns content from fixture file from __fixtures__ folder
+   * @param name name of the fixture file
+   * @param fixturesRoot is an optional, by default is current test folder
```

Comment:
```suggestion
   * @param fixturesRoot is an optional string, by default is current test folder
```

Reply:
```suggestion
   * @param {string} [fixturesRoot] - Where to find the fixtures, use the current test folder by default
```

I think this is the correct JSDoc syntax, but let me know if I'm doing it wrong!

The idea here is:
- Use `{string}` to say what type the input should be
- Wrap the paramater name in `[]` to mark it _optional_
- Use a hyphen `-` to mark the start of the `description` text

Relevant quotes from JSDocs docs [^docs]:

> **Name, type, and description, with a hyphen before the description**
> 
> ```js
> /**
>  * @param {string} somebody - Somebody's name.
>  */
> function sayHello(somebody) {
>     alert('Hello ' + somebody);
> }
> ```

> **An optional parameter (using JSDoc syntax)**
>
> 
> ```js
> /**
>  * @param {string} [somebody] - Somebody's name.
>  */
> function sayHello(somebody) {
>     if (!somebody) {
>         somebody = 'John Doe';
>     }
>     alert('Hello ' + somebody);
> }
> ```

[^docs]: https://jsdoc.app/tags-param.html

Reply:
The type is not required for typescript, as it's inferred from function declaration.

Reply:
```suggestion
   * @param [fixturesRoot] - Where to find the fixtures, uses the current test folder by default
```

Removed `{string}` from my suggestion, as we don't need it. 😉 

Will the rest of this suggestion work okay? Or is there something wrong with that as well? 🙈

Reply:
Fine bye me